### DISCUSSION TITLE
Integrating Native Google Sign-In with Supabase Authentication

### DISCUSSION THREAD

**Discussion by Hidayat1999** [supabase/supabase#16246] [Questions] [answered]
Currently, Supabase uses a web-based sign-in method, but I'd prefer to utilize the native Google Sign-In that doesn't launch a web browser. Instead, the native module prompt would appear directly within the app.

Is it possible to integrate the native sign-in module with Supabase's authentication in React Native App?

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/blog/native-mobile-auth
